data/text/trainer_class_names.inc

```diff
@@ -0,0 +1,107 @@
+	.string "{PKMN} TRAINER$  "
```

Comment:
Why don't you use this syntax?  `.string "AQUA LEADER", 13`

Reply:
Literally forgot that syntax exists haha